common.blocks/progressbar/progressbar.bemhtml

```diff
@@ -6,9 +6,15 @@ block('progressbar')(
 
     content()(
         function() {
+            var val = (this.ctx.val || 0) + '%';
+
             return {
                 elem : 'bar',
-                attrs : { style : 'width:' + (this.ctx.val || 0) + '%' }
+                attrs : {
+                    style : 'width:' + val,
+                    role : 'progressbar',
```

Comment:
For this we should change `style` and `aria-valuenow` attributes in different places. It will be strange.
For example:

```
<div class="progressbar" aria-valuenow="20">
    <div class="progressbar__bar" style="width:20%"></div>
</div>
```

Two DOM-nodes will be changed each second while some process will be going. If this is not so terrible, I can move these attributes